### PR TITLE
Add MatchingWordArray test

### PR DESCRIPTION
**Pull request by KeenanCronyn**
This adds a test for my MatchingWordArray function.

For the moment I'll continue to build successful tests, and then I can look at passing data that should fail and seeing if it does, but for the moment I just want to build out as much as I can.

### CODE REVIEW

**Comment by KeenanCronyn** [comment]
Let me know if you think that's a good idea or not...

File: src/WordscapesCheat.CLI/Program.cs [hunk 1]
```diff
@@ -27,9 +27,8 @@ static void Main(string[] args)
             //We can use the other args for other stuff later
             //e.g. word count or letter position
             string characterSet = args[0];
-            string validTest = "abc";
             List<string> dictionary = CheatFunctions.GetDictionary();
-            PrintMatchingWords(CheatFunctions.BuildMatchingWordsArray(validTest, dictionary));
+            PrintMatchingWords(CheatFunctions.BuildMatchingWordsArray(characterSet, dictionary));
```
Comment (Cronan) [inline src/WordscapesCheat.CLI/Program.cs]: I'd spotted this bug too :-)

File: src/WordscapesCheatTests/CheatFunctionsTests.cs [hunk 2]
```diff
@@ -19,5 +19,17 @@ public void BuildOccurenceArray_PassValidString_Succeed(string input, int[] expe
         {
             CollectionAssert.AreEqual(expected, CheatFunctions.BuildOccurenceArray(input));
         }
+
+        [DataTestMethod]
+        [DataRow("abc", "cab")]
```
Comment (Cronan) [inline src/WordscapesCheatTests/CheatFunctionsTests.cs]: This is exactly correct

File: src/WordscapesCheat.CLI/Program.cs (hunk 1 above)
Comment (KeenanCronyn) [reply src/WordscapesCheat.CLI/Program.cs]: Haha yes that’s what I get for making changes to it without properly documenting it

**Pull request merged**